test/specs/profiles/visibility-agg.js

```diff
@@ -77,7 +77,8 @@ describe('Test profile visibility', function () {
         console.log('user', user);
         promises.push(() => login(user));
         promises.push(() => MainPage.userMenu.click());
-        promises.push(() => MainPage.userMenu_myProfile.click());
+        promises.push(() => MainPage.userMenu_viewProfileLink);
+        promises.push((url) => browser.url(url));
         promises.push(() => browser.getUrl());
```

Comment:
Add return Promise.resolve(url) and get it through the params as line 83, then delete line 82